diff --git a/cmake/BTApplication.cmake b/cmake/BTApplication.cmake
index 33b00ea69..7e929609f 100644
--- a/cmake/BTApplication.cmake
+++ b/cmake/BTApplication.cmake
@@ -221,14 +221,6 @@ CONFIGURE_FILE("${CMAKE_CURRENT_SOURCE_DIR}/cmake/platforms/linux/bibletime.desk
 INSTALL(FILES "${CMAKE_CURRENT_BINARY_DIR}/bibletime.desktop"
         DESTINATION "${BT_DATAROOTDIR}/applications/" RENAME "info.bibletime.BibleTime.desktop")
 
-
-CONFIGURE_FILE("${CMAKE_CURRENT_SOURCE_DIR}/cmake/platforms/linux/info.bibletime.BibleTime.metainfo.xml"
-        "${CMAKE_CURRENT_BINARY_DIR}/info.bibletime.BibleTime.metainfo.xml" @ONLY)
-INSTALL(FILES "${CMAKE_CURRENT_BINARY_DIR}/info.bibletime.BibleTime.metainfo.xml"
-        DESTINATION "${BT_DATAROOTDIR}/metainfo/")
-
-
-
 IF(MSVC) # Windows:
   # sword locale information
   FILE(GLOB INSTALL_SWORD_LOCALE_LIST CONFIGURE_DEPENDS
