diff --git a/CMakeLists.txt b/CMakeLists.txt
index 3de56d97f..2031768f1 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -13,7 +13,7 @@ BLOCK(PROPAGATE PROJECT_VERSION)
   IF(IS_DIRECTORY "${CMAKE_CURRENT_SOURCE_DIR}/.git")
     FIND_PACKAGE(Git REQUIRED)
     EXECUTE_PROCESS(
-      COMMAND ${GIT_EXECUTABLE} describe --match "v*.*.*" --dirty --abbrev=12
+      COMMAND ${GIT_EXECUTABLE} describe --match "v*.*.*" --abbrev=12
       WORKING_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}
       OUTPUT_VARIABLE GIT_DESCRIBE
       ENCODING "UTF-8"
